unity-renderer/Assets/Textures/UI/Common/TexturesUI.spriteatlas

```diff
@@ -443,7 +443,7 @@ SpriteAtlas:
   - SpeakerOn
   - ContainerRadius6
   - VoiceChatCross
-  - DecentralandIsologotipo68
+  - DCLLogo
   - TopRect12
```

Comment:
it is so difficult to find this Atlas in the one big folder of assets. I would suggest add sub-folders for better visibility and clarity. For example .spriteatlas can be in the root UI/Common, while all atlased textures can be in sub-folder(s) depending on their grouping in the UI (as I see, not all of them are even so "Common" - EmotesOn/Off, VoiceChat. Favorites and etc)